bspi_iconmap.ini

```diff
@@ -0,0 +1,14 @@
+[Icons]
+dolphin = 
+chromium = 
+firefox = 
+signal = 
+slack = 
+spotify = 
+termite = 
+urxvt = 
+tilix = 
+alacritty = 
+kitty = 
+code-oss = 
+None =  
```

Comment:
Maybe since this is not pythonic `None` anymore, we could have a special catch-all key like `_other`?